### DISCUSSION TITLE
Feature Request: import chat logs

### DISCUSSION THREAD

**Discussion by helmut72** [element-hq/element-meta#497] [Ideas]
#### What would you like to do?

Import chat logs from an export.

#### Why would you like to do it?

I don't want lost chats from an old instalation/bad username. Like it's always possible for mails/Imap server.

A counterpart for https://github.com/vector-im/element-web/issues/2630

**Comment by daniellekirkwood** [comment]
This is marked as a discussion as we still need to discuss the product requirements and impacts around this request. It's cross-platform and impacts all parts of our product and product strategy.  Issues should be reserved for activities we believe are actionable or are on our backlog/roadmap. 

This is a process shift that Element is going through and so you'll see more and more issues become discussions as we work through our thousands of issues and requests. 

Please feel free to leave comments and suggestions on all Discussions as this helps us to review and prioritise them.

**Reply by helmut72** [reply]
It's possible to import chats form Slack:
https://element.io/blog/slack-migration/

... why isn't it possible to import exported chats from Element? I have still my bad configured (using subdomain instead of my domain) installation running from many years ago, because I don't want to lost my chats.
